docs/maintain-guides-democracy.md

```diff
@@ -158,6 +158,31 @@ when successful.
 
 ![democracy clear locks](assets/democracy_clear_locks.png)
 
+If you don not see an option to clear expired democracy votes, it may be that the lock is very old.
+You then must clear it by directly issuing the correct extrinsics.
+
+Navigate to the [Extrinsics page](https://polkadot.js.org/apps/#/extrinsics) and submit the
+following extrinsic: `democracy.removeVote(index)` using the account that you voted with. For the
+index number (ReferendumIndex), enter the number of the referendum for which you voted ("12" in the
+image below).
+
+You need to press the "Submit Transaction" button to submit the extrinsic.
+
+![democracy clear_lock_extrinsic_1](assets/democracy_clear_lock_extrinsic_1.png)
+
+Now submit the following extrinsic: `democracy.unlock(target)`, where target is your your account
+address.
+
+![democracy clear_lock_extrinsic_2](assets/democracy_clear_lock_extrinsic_2.png)
+
+If you return to the [Accounts page](https://polkadot.js.org/apps/#/accounts), you should see that
+the democracy lock has been released.
+
+Note that this applies only to locked DOT that were used for voting on referenda. In order to unlock
+DOT locked by voting for Polkadot Council members, you need to go to the
```

Comment:
What's the justification for capitalizing "Council"?

Reply:
I have always capitalized it.  Since there is only one, it seems like a proper noun.  Is this against our style guide?

Reply:
E.g., even though "court" is not a proper noun, I would write "the Supreme Court".

Reply:
Ok, that makes sense. I don't think it is in the style guide that why I'm asking. Maybe we can rephrase to "members of the Polkadot Council" since I think that sits better?